### PR TITLE
feat!: migrate to i18n 0.6.0

### LINKED ISSUE #10: Update @yandex-cloud/i18n package version
Hi, the `@yandex-cloud/i18n` api is not out of 0.x.x yet so npm semver resolution does not pay attention to hats `^`, so I was wondering if I could help you to migrate to new `@yandex-cloud/i18n` version `0.6.0`

### PR DESCRIPTION
**Pull request by ierehon1905**
Resolves #10

### CODE REVIEW

File: src/i18n/index.ts [hunk 1]
```diff
@@ -1,11 +1,18 @@
-import {i18n, I18N} from '../utils';
+import {i18n, setLang} from '../utils';
 
 import en from './en.json';
 import ru from './ru.json';
 
+export enum LANGS {
```
Comment (korvin89) [inline src/i18n/index.ts:6]: It should be a string union

**Review by korvin89 — CHANGES_REQUESTED** [review]

File: src/i18n/index.ts (hunk 1 above)
Comment (ierehon1905) [reply src/i18n/index.ts]: fixed

**Comment by jhoncool** [comment]
Let's add an example of using DashKit with i18n in README.md

File: src/i18n/index.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
-import {i18n, I18N} from '../utils';
+import {i18n, setLang} from '../utils';
 
 import en from './en.json';
 import ru from './ru.json';
 
+export type LANGS = 'ru' | 'en';
```
Comment (korvin89) [inline src/i18n/index.ts:6]: Could you rename this type to Lang?

Comment (ierehon1905) [reply src/i18n/index.ts]: renamed

**Comment by ierehon1905** [comment]
> Let's add an example of using DashKit with i18n in README.md

@jhoncool made a small example in readme 
image

**Review by korvin89 — APPROVED** [review]

**Review by jhoncool — APPROVED** [review]

**Pull request merged**